### DISCUSSION TITLE
dataLabels á la Highchart possible?

### DISCUSSION THREAD

**Discussion by U--d--o** [grafana/grafana#57826] [Dashboards, Panels & Visualizations]
With Highcharts it is possible to enable so-called dataLabels ($serie['dataLabels']['enabled'] = true;). 
Are these possibilities also available in Grafana and if so, how?
![Example Highcharts dataLabel](https://user-images.githubusercontent.com/5815527/198584457-25ef0ace-70fd-4baa-9ca2-a5009891c4b6.jpg)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
